vizro-core/docs/pages/cheatsheet/cheatsheet.html

```diff
@@ -951,219 +1257,179 @@ <h2 class="section-title">
+              <div class="card-title">
+                <a
+                  href="https://vizro.readthedocs.io/en/stable/pages/user-guides/data/#parametrize-data-loading"
+                  >Parametrized dynamic data</a
+                >
+              </div>
+              <div class="card-description">
+                Add arguments to your dynamic data loading function and control
+                it from the dashboard.
+              </div>
+              <div class="code-block-container">
+                <div class="code-block">
+                  <pre><code class="language-python">import pandas as pd
+from vizro.managers import data_manager
 
-        <!-- Actions & Customization Section -->
-        <section class="section">
-          <div class="section-header">
-            <h2 class="section-title">
-              <a
-                href="https://vizro.readthedocs.io/en/stable/pages/user-guides/actions/"
-                >Actions</a
-              >
-            </h2>
-          </div>
-          <div class="subsection-description">
-            Vizro actions are similar to callbacks in Dash.
-          </div>
-          <div class="one-column-grid">
-            <div>
-              <div class="subsection">
-                <div class="subsection-title">Export data</div>
-                <div class="subsection-description">
-                  To enable downloading data, you can add the
-                  <code>export_data</code> action function to the Button
-                  component.
-                </div>
-                <div class="code-block-container">
-                  <div class="code-block">
-                    <pre><code># Export data action
-from vizro.actions import export_data
+# Load data
+def load_iris_data(number_of_points=10):
+    iris = pd.read_csv("iris.csv")
+    return iris.sample(number_of_points)
 
-actions=[vm.Action(
-    function=export_data(),
-    ),
-]</code></pre>
-                  </div>
-                  <button class="copy-button" onclick="copyCode(this)">
-                    Copy
-                  </button>
-                </div>
-                <div class="subsection-title">Cross-filtering</div>
-                <div class="subsection-description">
-                  Cross-filtering enables you to click on data in one chart or
-                  table to filter other components in the dashboard.
-                </div>
-                <div class="code-block-container">
-                  <div class="code-block">
-                    <pre><code># Cross-filtering
-from vizro.actions import filter_interaction
+data_manager["iris"] = load_iris_data
 
-actions=[vm.Action(
-    function=filter_interaction(targets=["your-target-component-id"]),
-    ),
-]</code></pre>
-                  </div>
-                  <button class="copy-button" onclick="copyCode(this)">
-                    Copy
-                  </button>
+# Use in components
+vm.Graph(
+    figure=px.box(
+        "iris",
```

Comment:
Here and in the dynamic data example.
```suggestion
        data_frame="iris",
```